Sources/MagicSDK/Core/Relayer/URLBuilder.swift

```diff
@@ -13,27 +13,21 @@ import Foundation
 // Construct a URI with options encoded
 //
 public struct URLBuilder {
-    
+
     let encodedParams, url: String
     static let host = "https://box.magic.link"
 //  static let host = "http://192.168.0.106:3016"
```

Comment:
Should we remove ` //  static let host = "http://192.168.0.106:3016"` for cleanliness?

Reply:
Yes. I'll move this line to an onboarding guidebook for new members to set up the local environment